Example/healthvault-ios-sdk/ViewControllers/Timeline/MHVTimelineSnapshotViewController.m

```diff
@@ -0,0 +1,89 @@
+//
+// MHVTimelineSnapshotViewController.m
+// SDKFeatures
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "MHVTimelineSnapshotViewController.h"
+#import "MHVStatusLabel.h"
+#import "MHVTypeViewController.h"
+#import "MHVMoreFeatures.h"
+#import "MHVFeatureActions.h"
+#import "MHVStatusLabel.h"
+
+@interface MHVTimelineSnapshotViewController ()
+
+@property (nonatomic, strong) MHVActionPlanTasksResponseTimelineTask_ *timeline;
+@property (nonatomic, strong) MHVConnection *connection;
+
+@property (strong, nonatomic) IBOutlet UITextField *numberOfTasksLabel;
+@property (strong, nonatomic) IBOutlet MHVStatusLabel *statusLabel;
+
+@end
+
+@implementation MHVTimelineSnapshotViewController
+
+- (id)initWithTypeClass:(Class)typeClass useMetric:(BOOL)metric
+{
+    self = [super init];
```

Comment:
nit: We usually check if self is nil before setting properties on it. There is a rare possibility that the [super init] would return nil.

if (self)
{
    MHVConfiguration *config = MHVFeaturesConfiguration.configuration;
    self.connection = [[MHVConnectionFactory current] getOrCreateSodaConnectionWithConfiguration:config];
}